charts/stable/thelounge/values.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: thelounge/thelounge
+  pullPolicy: IfNotPresent
+  tag: 4.2.0-alpine
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the ${CHARTNAME} documentation
+# https://${CHARTNAME}.org/docs
```

Comment:
I think something went wrong here with the var substitution 🙂

Reply:
That or I generated the file wrong.

Ill fix